api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -111,153 +136,190 @@
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="2" style="padding-left: 1rem">
+      <div class="text bold-user-label">Your Username:</div>
+    </td>
+    <td colspan="3">
+      <div class="text user-info">${USERNAME}</div>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="2" style="padding-left: 1rem">
+      <div class="text bold-user-label">Your Temporary Password:</div>
+    </td>
+    <td colspan="3">
+      <div class="text user-info">${PASSWORD}</div>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <div class="completeRegistrationBtn">
+        <a class="button" href="${URL}">COMPLETE REGISTRATION</a>
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <div class="text section">
+        <div>
+          <div class="text bold-user-label" style="margin-bottom: 0.1rem">Step 1</div>
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Complete
+            Registration
           </div>
-          <div class="text">
-            <a class="text-link" href="https://support.google.com/chrome/answer/95346">Chrome</a>
-            <span> is the primary browser we currently support for the Researcher Workbench.
-            We highly recommend using Chrome for the best user experience.
-          </span>
+          <div class="stepCard step_rt_ct">
+            <ul style="padding-bottom: 1rem">
+              ${RT_STEPS}
+            </ul>
           </div>
         </div>
-      </td>
-    </tr>
-    <tr style="height: 1em"></tr>
-    <tr class="card">
-      <td colspan="3">
-        <div style="display: flex">
-          <div class="bullet">
-            <img src="${BULLET_2}">
+        <div style="padding-bottom: 0.6rem; padding-top: 1rem">
+          <div class="text bold-user-label" style="margin-bottom: 0.1rem">Step 2 (Optional)</div>
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Additional Data
+            Access
           </div>
-          <div class="text">
-            <span>
-              We recommend creating a <a class="text-link" href="https://support.google.com/chrome/answer/2364824">Chrome Profile </a>
-              with your ${ALL_OF_US} Researcher Workbench Google account. This will keep your Workbench browser sessions isolated if you use other Google accounts.
-            </span>
+          <div class="stepCard step_rt_ct">
+            <ul style="padding-bottom: 1rem">
+              ${CT_STEPS}
+            </ul>
           </div>
         </div>
-      </td>
-    </tr>
-    <tr style="height: 1em"></tr>
-    <tr class="card">
-      <td colspan="3">
-        <div style="display: flex">
-          <div class="bullet">
-            <img src="${BULLET_3}">
+        <div style="padding-bottom: 1.6rem">
+          <div class="text bold-user-label" style="margin-bottom: 0.1rem">Step 3</div>
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Sign the code
```

Comment:
casing inconsistency: `Sign the code of conduct` should be `Sign the Code of Conduct`.

Reply:
This is consistent with the DAR page header: 
**Step 3
Sign the code of conduct**

 Sign Data User Code of Conduct

Reply:
This's not a blocker. But I think DAR page header is wrong too.

Reply:
I agree, the previous comment was incomplete , what i meant was i can add both the changes (DAR page + email) in another PR